### ISSUE TITLE
script `fedbiomed_run` outputs assumes config is wrong but in reality another error occurred

### ISSUE THREAD

**Issue by srcansiz** [fedbiomed/fedbiomed#507]
In GitLab by @sharkovsky on Apr 17, 2023, 17:34

Currently [we do](https://gitlab.inria.fr/fedbiomed/fedbiomed/-/blob/develop/scripts/fedbiomed_run#L645) 
```bash
if ! forward_command_to_python_cli researcher "configuration create" "$IP_ADDRESS"; then
    echo -e "${RED}ERROR:${NC}"
    echo -e "${BOLD}Error related to configuration (config) file. Please check you have valid config file for researcher"
    exit 1
fi
```

But this leads to an incorrect error message printed to the user, in cases where something else triggered the error. This often happens on gpu nodes on nef, where the real error is something like
```
ImportError: /lib64/libstdc++.so.6: version `CXXABI_1.3.9' not found (required by /home/fcremone/.conda/envs/fedbiomed-node/lib/python3.9/site-packages/scipy/sparse/_sparsetools.cpython-39-x86_64-linux-gnu.so)
```

and can be solved by `module load gcc` (no relationship with config files)
